### PR TITLE
feat(monitoring): Modal result NLP interpretation

### PR DESCRIPTION
**Pull request by bouillonpi**
The NLP interpretation is displayed in a modal popping after submitting the NLP search-bar successfully.

### CODE REVIEW

File: frontend/src/app/search-bar.component.ts [hunk 1]
```diff
@@ -1,29 +1,41 @@
...
   ngOnInit() {
   }
 
+  openDialog(response) {
+    this.dialogRef = this.dialog.open(SearchResultDialog);
+
+    this.dialogRef.componentInstance.response = response;
+
+    this.dialogRef.afterClosed().subscribe(result => {
+      console.log('result: ' + result);
```
Comment (cassand) [inline frontend/src/app/search-bar.component.ts]: We can interpret the result to give the user a chance to cancel the request

File: frontend/src/app/search-result.component.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import {Component, Input} from "@angular/core";
+import {MdDialogRef} from "@angular/material";
+
+@Component({
+  selector: 'search-result-dialog',
+  templateUrl: './search-result.component.html'
+})
+
```
Comment (FunkySayu) [inline frontend/src/app/search-result.component.ts]: Remove blank line for consistency

File: frontend/src/app/search-result.component.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+export class SearchResultDialog {
+
+  @Input() response: any;
+
+  constructor(public dialogRef: MdDialogRef<SearchResultDialog>) { }
+
+  ngOnInit() {
+    console.log(this.response)
+  }
```
Comment (FunkySayu) [inline frontend/src/app/search-result.component.ts:16]: Add blank line for consistency

File: frontend/src/search-result.css [hunk 4]
```diff
@@ -0,0 +1,7 @@
+.search-result-dialog {
```
Comment (FunkySayu) [inline frontend/src/search-result.css:1]: [Open Discussion, Not todo in this pull request]
There're a lot of css files in the src/ dir. Shouldn't this be migrated to another directory (e.g. `src/css/`) for sanitizing `src/` ?

File: frontend/src/app/search-bar.component.ts [hunk 5]
```diff
@@ -1,29 +1,41 @@
...
   constructor(private querySender: QuerySenderService,
-              private mdSnackBar: MdSnackBar) {
+              private mdSnackBar: MdSnackBar,
+              public dialog: MdDialog) {
   }
 
   ngOnInit() {
   }
 
+  openDialog(response) {
```
Comment (FunkySayu) [inline frontend/src/app/search-bar.component.ts:28]: Please document this

File: frontend/src/app/search-bar.component.ts (hunk 1 above)
Comment (FunkySayu) [inline frontend/src/app/search-bar.component.ts]: Looks like debugging output

File: frontend/src/app/search-bar.component.ts [hunk 6]
```diff
@@ -34,4 +46,9 @@ export class SearchBarComponent implements OnInit {
     this.mdSnackBar.open(logStr, 'Close');
   }
 
+  successHandler(response: any) {
```
Comment (FunkySayu) [inline frontend/src/app/search-bar.component.ts]: You don't really need this function

File: frontend/src/app/search-result.component.html [hunk 7]
```diff
@@ -0,0 +1,30 @@
+<section class="search-result-dialog">
+
+  <h1>NLP Interpretation</h1>
+
+  <table *ngIf="response" class="bordered striped centered-th">
+    <thead>
+    <tr>
```
Comment (FunkySayu) [inline frontend/src/app/search-result.component.html]: Missing indentation level

File: frontend/src/app/search-result.component.html [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+      <th *ngIf="response.status">Status</th>
+      <th *ngIf="response.place">Place</th>
+      <th *ngIf="response.from">From</th>
+      <th *ngIf="response.to">To</th>
+      <th *ngIf="response.type">Type</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
```
Comment (FunkySayu) [inline frontend/src/app/search-result.component.html]: Same as above

**Review by FunkySayu — COMMENTED** [review]
Nits. Once fixed, LGTM

File: frontend/src/search-result.css (hunk 4 above)
Comment (bouillonpi) [reply frontend/src/search-result.css:1]: Definitely. I will sanitize this after merging feature/monitoring-feedback and feature/monitoring-tasks on feature/monitoring.

**Review by cassand — APPROVED** [review]

**Pull request merged**